tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/RabbitMQ.cs

```diff
@@ -176,7 +176,7 @@ private string BasicGetMessage(string queueName)
 
             // See this project's .csproj file for target framework => RabbitMQ.Client mappings
             // 'netcoreapp3.1' is tied to RabbitMQ.Client version 6.x+, making this necessary
```

Comment:
may want to add that NET48 is also using 6.2.1, or genericize the language so the comment doesn't need to be updated for each change

Reply:
I reorganized things so that we `#define` a `RABBIT6PLUS` symbol conditionally based on the target framework at the top of the file, and then use the symbol in the `#if` directives down in the actual code.  Let me know if you think that helps to clarify what is going on.